### ISSUE TITLE
Unable to upload image after upgrade to laravel 9

### ISSUE THREAD

**Issue by farheen0702** [thedevdojo/voyager#5789]
### Laravel version

9

### PHP version

8.0

### Voyager version

1.6

### Database

8.0

### Description

![image](https://github.com/thedevdojo/voyager/assets/33607209/0ef83a64-d6e9-477b-9730-2114f908b03a)

I am facing issue on view of my edit pages that this image upload is calling the wrong api and i'm not able locate the exact the issue and from where can we update this.

### Steps to reproduce

Body component has option for Multiple function and one of them image upload
![image](https://github.com/thedevdojo/voyager/assets/33607209/bb05f78d-d565-4295-b39a-40e1cc205c68)

### Expected behavior

The api should be upload whereas it's calling edit
The same code is working for another project

### Screenshots

![image](https://github.com/thedevdojo/voyager/assets/33607209/5fd9156a-0cc5-4b31-87c9-856538201a6e)

**Comment by ricidelphy** [comment]
yes, I experienced the same thing as you, is there someone who can solve this bug?

**Comment by ihzadeprian1305** [comment]
Same experience after using voyager 1.6 :(, before that i used voyager 1.x-dev and everything was okay, any idea guys?
